pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -50,11 +53,17 @@ const (
 	burstingThroughputMode = awsefs.ThroughputModeBursting
 	defaultThroughputMode  = burstingThroughputMode
 
-	efsType            = "EFS"
-	k10BackupVaultName = "k10vault"
-	testMarker         = ""
+	efsType                   = "EFS"
+	defaultK10BackupVaultName = "k10vault"
+	testMarker                = ""
 
 	maxRetries = 10
+
+	efsBackupVaultNameEnv = "EFS_BACKUP_VAULT_NAME"
+)
+
+var (
+	efsBackupVaultName = getEnvAsStringOrDefault(efsBackupVaultNameEnv, defaultK10BackupVaultName)
```

Comment:
@bathina2 - can we add this to the config map - see `NewEFSProvider` instead of environment variables?

Reply:
@vkamra This will be set in the config map and referred to as an environment variable. See https://github.com/kastenhq/k10/pull/13188

Reply:
Oh you mean as an input to create the Provider.

Reply:
> Oh you mean as an input to create the Provider.

Yep 👍